source/js/buyers-guide/components/donate-modal/donate-modal.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import ReactGA from 'react-ga';
+import DNT from '../../dnt.js';
+
+
+export default class DonateModal extends React.Component {
+  constructor(props) {
+    super(props);
+    let dismissed = !!sessionStorage.getItem('dismissed') || false;
+    this.state = {
+      delay: this.props.delay || 100,
+      visible: false,
+      dismissed
+    };
+  }
+
+  componentDidMount() {
+    if (this.state.dismissed) {
+      this.props.onDismiss();
+    }
+    setTimeout(e => this.setState({ visible: true }), this.state.delay);
+  }
+
+  dismiss() {
+    sessionStorage.setItem('dismissed', 'dismissed');
+    this.setState({
+      dismissed: true
+    });
+  }
+
+  trackDonation() {
+    if (DNT.allowTracking) {
+      ReactGA.event({
+        category: `site`,
+        action: `donate tap`,
+        label: `donate popup`
+      });
+    }
+  }
+
+  render() {
+    if (this.state.dismissed) {
+      return null;
+    }
+
+    return (
+      <div className={`donate-modal ${this.state.visible ? 'show' : ''}`}>
```

Comment:
to be honest, we could even hard-code this in the base template, but that makes the interaction a bit more work in bg-main.js. Thoughts, @alanmoo ?

Reply:
No specific opinions here, one way or the other. I'd say optimize for browser performance; I noticed pages choking up a bit before the user could scroll when loading on older phones.